client/src/model/store_plugin.ts

```diff
@@ -15,7 +15,6 @@ const createRedux = (store: EnhancedStore) => {
 
       store.subscribe(() => {
         rootStore.state = store.getState();
-        console.log(`The state has been updated ${JSON.stringify(rootStore.state)}`);
```

Comment:
This is very noisy

Reply:
+1. Though I'm up to still log some stuff that's useful.